File: C/1833-find-the-highest-altitude/find-the-highest-altitude.py
class Solution(object):
    def largestAltitude(self, gain):
        """
        :type gain: List[int]
        :rtype: int
        """
        li={}
        li[0]=0
        for i in range(0,len(gain)):
            li[i+1]=li[i]+gain[i]
        gainSize=len(gain)
        i=0
        max=-100
        while(i<gainSize+1):
            if(li[i]>max):
                max=li[i]
            i+=1
        return max

File: C/1833-find-the-highest-altitude/find-the-highest-altitude.c
int largestAltitude(int* gain, int gainSize) {
    int temp[gainSize+1];
    temp[0]=0;
    for(int i=0;i<gainSize;i++){
        temp[i+1]=temp[i]+gain[i];
    }
    // for(int i=0;i<gainSize+1;i++){
    //     printf("%d ",temp[i]);
    // }
    int i=0,max=-100;
    while(i<gainSize+1){
        if(temp[i]>max){
            max=temp[i];
        }
        i++;
    }
    return max;
}